src/routes/+layout.server.ts

```diff
@@ -59,33 +65,61 @@ export const load: LayoutServerLoad = async ({ locals, depends }) => {
 
 	const loginRequired = requiresUser && !locals.user && userHasExceededMessages;
 
+	const disableAssistants = ENABLE_ASSISTANTS !== "true";
```

Comment:
to go with [here](https://github.com/huggingface/chat-ui/pull/639/files#r1445950232)
```suggestion
	const enableAssistants = ENABLE_ASSISTANTS === "true";
```